pkg/vault/helpers.go

```diff
@@ -94,3 +98,46 @@ func ReadBasicAuth(client Client, path string) (*config.BasicAuth, error) {
 		Password: passwordStr,
 	}, nil
 }
+
+// ToURI constructs a vault: URI for the given path and key
+func ToURI(path string, key string) string {
+	return fmt.Sprintf("vault:%s:%s", path, key)
+}
+
+// ReplaceURIs will replace any vault: URIs in a string, using the vault client
+func ReplaceURIs(s string, client Client) (string, error) {
```

Comment:
I would be nice to have some unit tests for this function.

Reply:
I did do a test but for the helm func, I'll copy it in here.